### PR TITLE
chore: aspnet integration docs razor fixes

### PR DESCRIPTION
**Pull request by iantonge**
# Description

fix razor syntax in aspnet integration documentation

## Motivation & context
There are a couple of minor mistakes in the razor syntax on the aspnet integrations documentation:

Copying the CDN link from the aspnet integration docs into a razor template causes a compilation error as the `@` symbol needs to be escaped in razor.

Copying the fast-card sample code from the documentation renders a card where the title is the literal text 'ViewData["Title"]' rather than the value which should be 'Home Page'.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

**Comment by iantonge** [comment]
I'm not entirely clear on the etiquette of creating a pull request, or if I should create an issue first. For a couple of small changes to the docs an issue might not be necessary, but I'm happy to take direction on that.

As an aside, the link to [https://www.fast.design/docs/en/contributing/standards](https://www.fast.design/docs/en/contributing/standards) appears to just show me that same content as [https://www.fast.design](https://www.fast.design) right now.

**Comment by EisenbergEffect** [comment]
@iantonge This is perfect. Thank you! No need to create an issue for a doc fix like this. We'll get this reviewed and merged soon.

The contributor guide is here: https://www.fast.design/docs/community/contributor-guide Where did you find that old link? I'd like to get that fixed if you recall where you found it.

File: packages/web-components/fast-foundation/docs/integrations/aspnet.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Now that we've got our basic project setup, we need to add our web components sc
 To add a CDN script for `fast-components` use the following markup:
 
 ```html
-<script type="module" src="https://unpkg.com/@microsoft/fast-components"></script>
+<script type="module" src="https://unpkg.com/@@microsoft/fast-components"></script>
```
[the remark this answers is not in the document]
Comment (EisenbergEffect) [reply packages/web-components/fast-foundation/docs/integrations/aspnet.md:29]: The Blazor syntax treats `@` as a special character. So, if you need to use a `@` you have to double it up in order to escape it.

**Comment by iantonge** [comment]
@EisenbergEffect the old link is in the "Process & policy checklist" section of the PR template.

File: packages/web-components/fast-foundation/docs/integrations/aspnet.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (chrisdholt) [reply packages/web-components/fast-foundation/docs/integrations/aspnet.md:29]: Interesting! I would not have guessed that would apply to strings like this. Learning new things!

**Pull request merged**